src/test/scala/com/microsoft/hyperspace/index/IndexManagerTests.scala

```diff
@@ -51,7 +51,7 @@ class IndexManagerTests extends SparkFunSuite with SparkInvolvedSuite {
       .write
       .parquet(sampleParquetDataLocation)
 
-    df = spark.read.parquet(sampleParquetDataLocation)
+    df = spark.read.option("path", sampleParquetDataLocation).parquet(sampleParquetDataLocation)
```

Comment:
What's the reason for adding this `option` here? I think the new tests you are adding should cover this scenario?

Reply:
Fixed